media/css/mozorg/home/home.scss

```diff
@@ -472,7 +472,7 @@ main {
         padding-top: 40px;
 
         &:before {
-            @include at2x('/media/img/firefox/common/firefox-logo.png');
+            @include at2x('/media/img/firefox/template/logo.png');
```

Comment:
Yeah, the logo.png looks fuzzy for me too. That's why I added the high-res version in the first place :smile: